novawallet/Modules/WalletsList/Selection/WalletSelectionInteractor.swift

```diff
@@ -42,10 +70,34 @@ extension WalletSelectionInteractor: WalletSelectionInteractorInputProtocol {
             switch result {
             case .success:
                 self?.eventCenter.notify(with: SelectedAccountChanged())
-
                 self?.presenter?.didCompleteSelection()
             case let .failure(error):
-                self?.presenter?.didReceiveError(error)
+                self?.presenter?.didReceive(saveError: error)
+            }
+        }
+    }
+
+    func updateWalletsStatuses() {
```

Comment:
Le't move application of these changes to the sync service. In this case running of these operation will not depend on the interactor lifecycle.